src/spell_mem.cpp

```diff
@@ -575,6 +620,7 @@ void done_memming(CharData *ch) {
 }
 
 void rem_memming(CharData *ch) {
+    char_printf(ch, "You have recovered all your spell slots.\n");
```

Comment:
Other things might kick you out of the mem state.  (Getting attacked for example.)
We should probably move this to the check to see if there are no more filled slots.

Reply:
rem_memming was only called in a few places, mostly in the memming_event function controller.  However, we never go into that function now.  We need to add a place where we remove the meditate flag if the player gets attacked or becomes drunk.